src/lib/pickles_types/plonk_types.ml

```diff
@@ -1358,8 +1595,8 @@ module Messages = struct
         ~dummy_group_element:dummy ~bool
     in
     let lookup =
-      Lookup.opt_typ Impl.Boolean.typ ~uses_lookup ~lookups_per_row_4
-        ~runtime_tables ~dummy:[| dummy |]
+      Lookup.opt_typ Impl.Boolean.typ ~uses_lookup:uses_lookups
```

Comment:
Note for later: use the same names for the function parameter and for the field name wrt `uses_lookup(s)`